src/sentry/api/endpoints/organization_index.py

```diff
@@ -262,7 +262,7 @@ def post(self, request: Request) -> Response:
             if result.get("agreeTerms"):
                 terms_accepted.send_robust(
                     user=request.user,
-                    organization=org,
+                    organization_id=org.id,
```

Comment:
Has the receiver in getsentry been updated for this change in parameters yet? There are also a few senders of this signal in getsentry that will need to be updated.

Reply:
Yup, that's why I expanded the receiver in this PR -- you'll notice it can now take either organization or organization id, and it works both ways, until I can remove organization usage in getsentry in a future PR.

Reply:
Should we still send organization to ensure that CI doesn't have a failing build if it manages to start between the two pull requests being merged?